test/boost/mutation_test.cc

```diff
@@ -710,6 +710,11 @@ SEASTAR_TEST_CASE(test_cell_ordering) {
         atomic_cell::make_live(*bytes_type, 1, bytes("value")),
         atomic_cell::make_live(*bytes_type, 1, bytes("value"), expiry_1, ttl_1));
 
+    testlog.debug("Live cells are ordered before expiring cells, regardless of their value");
```

Comment:
"live" cells does not contrast with "expiring" cells. It seems that in your example, both cells are live - just one has a ttl and one doesn't.

Reply:
ok, will clarify message.

Reply:
Oops, I changed similar comments but missed this one...
Will fix.

Reply:
fixed in next version (f298a9e485)